### PR TITLE
ScaleScore should allow values to be null 

### PR DESCRIPTION
**Pull request by malaffoon**
Not all scores will have values and/or levels set. 

I ran this locally and everything seemed to work fine with ELPAC claims, which have no score/std-err.

### CODE REVIEW

File: reporting-service/src/test/java/org/opentestsystem/rdw/reporting/TestData.java [hunk 1]
```diff
@@ -429,15 +429,15 @@
                 .sessionId("session5")
                 .dateTime(Instants.from("1997-01-05"))
                 .scaleScore(
-                        ScaleScore.builder().level(2).value(2400).standardError(24).build()
+                        ScaleScore.builder().level(2).value(2400).standardError(24.0).build()
                 )
                 .claimScaleScores(newArrayList(
-                        ScaleScore.builder().level(1).value(1200).standardError(120).build(),
-                        ScaleScore.builder().level(2).value(2200).standardError(220).build(),
-                        ScaleScore.builder().level(3).value(3200).standardError(320).build()
```
Comment (malaffoon) [inline reporting-service/src/test/java/org/opentestsystem/rdw/reporting/TestData.java]: I tweaked one of the test exams to have null values for claims.

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 2]
```diff
@@ -973,7 +973,7 @@
       },
       "instructional-resources": {
         "button": "Instructional Resources",
-        "description": "Upload instructional resources in the system."
+        "description": "Upload links to instructional resources in the system."
```
Comment (malaffoon) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: unrelated change from SB feedback

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/model/ScaleScore.java [hunk 3]
```diff
@@ -5,9 +5,9 @@
  */
 public class ScaleScore {
 
-    private int level;
-    private int value;
-    private double standardError;
+    private Integer level;
+    private Integer value;
+    private Double standardError;
```
Comment (esotrope) [inline common/src/main/java/org/opentestsystem/rdw/reporting/common/model/ScaleScore.java]: is it possible for some of the values to be null and not all? i think before we opted for an absent/null entry in an array to represent all of these values being null.

File: webapp/src/main/webapp/src/assets/i18n/en.json (hunk 2 above)
Comment (esotrope) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: 👍

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/model/ScaleScore.java (hunk 3 above)
Comment (malaffoon) [reply common/src/main/java/org/opentestsystem/rdw/reporting/common/model/ScaleScore.java]: Yes, that functionality is still there (handled in the service code): if all the values are missing a null value is stuffed in the ScaleScore list.
